### ISSUE TITLE
2.10.0 stable release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3383]
- #3376 

## Pick list
```
git cherry-pick 641d8e2851908afe5f8fa124c8bbf8810a09fe6b
```

**Comment by joeparsons** [comment]
## Release notes:

```
## New changes since 2.10.0-alpha2
- Close #3375 Install AZ Finder module if AZ Demo module is already installed. (#3376)

## Important notes
- **Important**: [Database updates](https://github.com/az-digital/az_quickstart/wiki/Maintaining-Quickstart-2-Sites#applying-database-updates) and [Distribution configuration updates](https://github.com/az-digital/az_quickstart/wiki/Maintaining-Quickstart-2-Sites#applying-distribution-configuration-updates) should _always_ be applied when updating Quickstart sites.
- **Important**: Updating from a Quickstart codebase less than 2.6.x is no longer supported.
- To update to a Quickstart 2.10.x release (e.g. 2.10.0) from a version less than 2.6.8 you must first update to 2.6.8 or greater and run database updates. Database updates added to Quickstart prior to version 2.6.0 have been removed in 2.7.x, 2.8.x and 2.9.x and some core modules included in Drupal 9 have been removed in Drupal 10.
- **For Site Updaters Managing Their Own Environments**: With the inclusion of a minor Drupal update in this release, it's imperative for site updaters who manage their own environments to thoroughly review the Drupal release notes and their own setups.   Please review https://github.com/az-digital/az-quickstart-scaffolding for the latest recommendations.
- **Quickstart 2.8.x will no longer receive security support. Sites on Quickstart 2.8.x or earlier should upgrade to a supported release as soon as possible**. See our [release policy](https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#release-overview) for more information.
- For more relevant info about upgrading from 2.6.x see [Quickstart 2.7.0 Upgrade Notes](https://github.com/az-digital/az_quickstart/wiki/Quickstart-2.7.0-Upgrade-Notes).

## Major changes (not included in 2.9.x)
- Closes #3334 Remove modules deprecated as of 2.8.x. (#3336)

### Deprecated modules slated for removal

## All changes not included in 2.9.x

### Bugfix

### Breaking or Visual Change to Existing Behavior

### Drupal Contrib Project Updates

### Editor Experience

### Developer Experience

### Other

## Changes also included in 2.9.x

### Bugfix

### Breaking or Visual Change to Existing Behavior

### Drupal Contrib Project Updates

### Editor Experience

### Developer Experience
```

**Comment by trackleft** [comment]
[Version 2.10 Released](https://github.com/az-digital/az_quickstart/releases/tag/2.10.0) 🎉
